teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -139,7 +145,9 @@ public void run() {
     try {
       initialise();
       if (confirmationEnabled) {
-        confirmExits();
+        if (!confirmExits()) {
+          return;
```

Comment:
We lost the return code in this change.  We should be returning 1 here so we get a non-zero exit code.  Class would implement `Callable<Integer>` rather than `Runnable` as well and return 0 where it currently just exits the function without calling System.exit.